tests/app_test.py

```diff
@@ -43,7 +43,7 @@ def test_index_tsv():
     with boddle(params={'row_count':'5', 'offset':'0'}):
         res = bottle_app.index_tsf()
         lines = res.split('\n')
-        assert len(lines) == 6
+        assert len(lines) == 7
```

Comment:
Ugh. My `assert len(lines)==6` was based on the idea that there were 5 lines. However, a line terminated by `\n` will result in an empty:
```
>>> "this\n".split('\n')
['this', '']
>>>
```